content/sensu-go/6.7/observability-pipeline/observe-schedule/checks.md

```diff
@@ -560,6 +560,120 @@ Learn how to use check tokens with the [Sensu tokens reference documentation][5]
 **NOTE**: Check tokens are processed before check execution, so token substitutions will not apply to check data delivered via the local agent socket input.
 {{% /notice %}}
 
+## Subdues
+
+Use the [`subdues` attribute][75] in check definitions to set specific periods of time when Sensu will not send alerts based on the events the check produces.
+Subdues allow you to schedule alert-free periods of time, such as during sleeping hours, weekends, or special maintenance periods.
+
+You can set more than one subdue at a time.
+Each subdue includes a begin and end time as well as how often to repeat the subdue, if desired.
+
+For example, this check would be subdued from 5:00 p.m. until 8:00 a.m. PDT on every weekday, and for the entire day on weekends, as well as every Friday from 10:00 until 11:00 a.m. PDT:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check_cpu
+spec:
+  command: check-cpu-usage -w 75 -c 90
+  interval: 60
+  handlers:
+  - slack
+  publish: true
+  round_robin: false
+  runtime_assets:
+  - check-cpu-usage
+  subdues:
+  - begin: "2022-04-18T17:00:00-07:00"
+    end: "2022-04-19T08:00:00-07:00"
+    repeat:
+    - weekdays
+  - begin: "2022-04-23T00:00:00-07:00"
+    end: "2022-04-23T23:59:59-07:00"
+    repeat:
+    - weekends
+  - begin: "2022-04-22T10:00:00-07:00"
+    end: "2022-04-22T11:00:00-07:00"
+    repeat:
+    - friday
+  subscriptions:
+  - system
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "check_cpu"
+  },
+  "spec": {
+    "command": "check-cpu-usage -w 75 -c 90",
+    "interval": 60,
+    "handlers": [
+      "slack"
+    ],
+    "publish": true,
+    "round_robin": false,
+    "runtime_assets": [
+      "check-cpu-usage"
+    ],
+    "subdues": [
+      {
+        "begin": "2022-04-18T17:00:00-07:00",
+        "end": "2022-04-19T08:00:00-07:00",
+        "repeat": [
+          "weekdays"
+        ]
+      },
+      {
+        "begin": "2022-04-23T00:00:00-07:00",
+        "end": "2022-04-23T23:59:59-07:00",
+        "repeat": [
+          "weekends"
+        ]
+      },
+      {
+        "begin": "2022-04-22T10:00:00-07:00",
+        "end": "2022-04-22T11:00:00-07:00",
+        "repeat": [
+          "friday"
+        ]
+      }
+    ],
+    "subscriptions": [
+      "system"
+    ]
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+### Subdues and repeat
+
+If you include a `repeat` array in a `subdues` object, Sensu will start the subdue period on the date you specify.
+After the first subdue period, Sensu uses the `begin` and `end` values only to determine the time of day to start and stop the subdue.
+
+In the above example, on April 19, 2022, Sensu will apply the weekday subdue at 5:00 p.m. PDT and end it on April 20 at 8:00 a.m. PDT.
+On April 20, Sensu will apply the weekday subdue again at 5:00 p.m. PDT and end it on April 21 at 8:00 a.m. PDT, and so on.
```

Comment:
Subdues will **not** extend into the past. For example suppose `begin` is set to `2023-01-01T14:00:00-0400` and `end` to `2023-01-01T15:00:00-0400` with a `repeat` of `mondays`.

The check will be subdued every Monday between 2pm and 3pm with a time offset of 4 hours behind UTC. However since `begin` is set to Jan 1st 2023 the subdue logic will not be applied before then (i.e. the check will run normally throughout 2022).